### PR TITLE
doc: Move COC proposal to stage 2

### PR DESCRIPTION
**Pull request by mhdawson**
I removed a number of the questions as I think the last iteration answered them.  Please let me know if they are still open and need to be added back in.

### CODE REVIEW

**Comment by mhdawson** [comment]
@openjs-foundation/cpc please review/comment.  Would be good to see if there is any discussion that needs to take place before it moves to stage 2.

File: proposals/stage-2/CODE_OF_CONDUCT/README.md [hunk 1]
```diff
@@ -37,20 +37,10 @@ foundation.
...
+### Other thoughts to consider later on
 * Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email. TODO: copy for a form.
 * Repo: openjs/moderation (not provisioned). As commented in PR review, hosting sensitive personal information on GitHub needs due consideration, review, and possibly legal consultation.
-
-### What to look forward to
 * Beacon project
-* Code of Conduct Project: A cross-project collaboration between multiple open source tech communities RxJS, AngularJS, Node.js, Vue.js etc to find consensus on Code of Conducts. Progress is rapid and can add definition to our own process (where possible) within 6 months. Recommend incorporating the findings and avoid duplication of effort.
\ No newline at end of file
+* Code of Conduct Project: A cross-project collaboration between multiple open source tech communities RxJS, AngularJS, Node.js, Vue.js etc to find consensus on Code of Conducts. Progress is rapid and can add definition to our own process (where possible) within 6 months. Recommend incorporating the findings and avoid duplication of effort.
```
Comment (boneskull) [inline proposals/stage-2/CODE_OF_CONDUCT/README.md]: > A cross-project collaboration between multiple open source tech communities (RxJS, AngularJS, Node.js, Vue.js,  etc.)

**Comment by bnb** [comment]
Can't comment on the line but the doc still says `Stage 1` on line 2.

**Comment by mhdawson** [comment]
@bnb thanks for catching that, fixed.

**Comment by mhdawson** [comment]
@boneskull updated.

**Pull request merged**